text/0000-match-ergonomics.md

```diff
@@ -122,30 +126,40 @@ than using auto-deref and autoreferencing, however, this proposal introduces
 the idea of "reference inversion". Reference inversion would allow references
 to structures to be matched as though they were structures of references.
 
+I'm not sure if it is useful to cause this reference inversion - it seems a bit like jargon to me. "Default binding modes" (used in the intro) sounds better to me
```

Comment:
Yeah, I agree. That's an  accidental leftover from when I was referencing the original Stebalien proposal, rather than the one Niko drew up. I'll fix that.